README.md

```diff
@@ -5,10 +5,11 @@ The autoloader for autoloading classes, enums, functions, typedefs, and constant
 Usage
 =====
 
-1. Add an `hh_autoload.json` file (see section below) and optionally remove your configuration from composer.json
+1. Add an `hh_autoload.json` file (see section below)
 2. `composer require hhvm/hhvm-autoload`
-3. Replace any references to `vendor/autoload.php` with  `vendor/autoload.hack` and call `Facebook\AutoloadMap\initialize()`
-4. To re-generate the map, run `vendor/bin/hh-autoload`, `composer dump-autoload`, or any other command that generates the map
+3. Require the autoload file from your entrypoint functions using `require_once __DIR__ . '/..(repeat as needed)/vendor/autoload.hack`
```

Comment:
while you're here, parens :)

Reply:
Added parens both for the require_once statement and grouping the `/..`. I didn't know which one you meant.